data/fault_drawer.bss.s

```diff
@@ -12,7 +12,7 @@
 .balign 16
 
 glabel sFaultDrawer
-    .space 0x3C
+    .space 0x40
 
-glabel D_8016B6BC
-    .space 0x24
+glabel D_8016B6C0
+    .space 0x20
```

Comment:
one of the things I don't get is `sFaultDrawer` is 0x3C bytes so why change to 0x40

the changes seem to go from bss size = 0x3C+0x24=0x60 to 0x40+0x20=0x60 ? am I missing the assembler aligning the symbols somehow?

Reply:
I changed this to be in sync with the code, which now reads:

```
FaultDrawer sFaultDrawer;
char D_8016B6C0[0x20];
```

where the distance between `sFaultDrawer` and `D_8016B6C0` is 0x40 bytes (0x3C bytes for the struct and 0x4 bytes of padding). What about something like this?

```
glabel sFaultDrawer
    .space 0x3C
    .space 0x4  # padding

glabel D_8016B6C0
    .space 0x20
```

Reply:
oh yeah that would be clearer
not sure why there's padding there :thinking:

Reply:
yeah, me neither